src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -41,6 +41,10 @@ def notifiable_link_text
       else
         "Removed as #{role} of a project"
       end
+    when 'Event::AddedUserToGroup'
+      "You got added to group '#{@notification.event_payload['group']}'"
+    when 'Event::RemovedUserFromGroup'
+      "You got removed from the group '#{@notification.event_payload['group']}'"
```

Comment:
```suggestion
      "#{event_payload['who'] || Someone} added you to the group '#{@notification.event_payload['group']}'"
```

Reply:
Changed with "removed you from"

Reply:
Same here, remove the description as it is duplicated code.
https://github.com/openSUSE/open-build-service/pull/16242#discussion_r1636248476